index.html

```diff
@@ -69,11 +69,40 @@ <h2>Bust IT: Cancer Without Borders</h2>
         <main id="home" class="footer-allowance">
             <h1 class="readable-width text-center title">Bust IT - Cancer Without Borders</h1>
 
+            <section id="user-utils" class="section-margin">
```

Comment:
@auxfuse This id needs to stay as #awareness so that the navbar link to it works